src/common/templates/includes/macros.njk

```diff
@@ -24,13 +71,11 @@
 {% macro submit_buttons(analyticsPage, csrfToken, __) %}
   <input id="save-and-continue" class="govuk-button" type="submit"
     name="buttonClicked"
-    value="{{__('form_save_and_continue')}}"
-    onclick="sendAnalytics(event, '{{ analyticsPage }} - Save and Continue', 'click')">
+    value="{{__('form_save_and_continue')}}">
```

Comment:
Very minor. I think it's better to add a space after the opening brackets and before the closing brackets. So, `{{ __('form_save_and_continue') }}`.

Reply:
Changes made.